miner/worker.go

```diff
@@ -1404,16 +1406,27 @@ func (w *worker) getSimulatedBundles(env *environment) ([]types.SimulatedBundle,
 		return nil, nil
 	}
 
-	bundles := w.eth.TxPool().MevBundles(env.header.Number, env.header.Time)
+	bundles, ccBundlesCh := w.eth.TxPool().MevBundles(env.header.Number, env.header.Time)
 
 	// TODO: consider interrupt
 	simBundles, err := w.simulateBundles(env, bundles, nil) /* do not consider gas impact of mempool txs as bundles are treated as transactions wrt ordering */
 	if err != nil {
-		log.Error("Failed to simulate flashbots bundles", "err", err)
+		log.Error("Failed to simulate bundles", "err", err)
 		return nil, err
 	}
 
-	return simBundles, nil
+	ccBundles := <-ccBundlesCh
```

Comment:
Nitpick: I wouldn't separate the simulation of bundles here, as long as I don't return simBundles & simCcBundles separately. But I would add a log to give an insight on the size of simCcBundles.

Reply:
The bundles coming from the channel are not ready when we start the simulation of the other bundles, it can take up to few ms for the channel to be ready.